app/src/gplay/java/com/nextcloud/talk/services/firebase/MagicFirebaseMessagingService.kt

```diff
@@ -162,6 +162,10 @@ class MagicFirebaseMessagingService : FirebaseMessagingService() {
                             cancelExistingNotificationWithId(applicationContext, signatureVerification!!.userEntity, notificationId)
                         } else if (deleteAll) {
                             cancelAllNotificationsForAccount(applicationContext, signatureVerification!!.userEntity)
+                        } else if (deleteMultiple) {
+                            notificationIds.forEach {
+                                cancelAllNotificationsForAccount(applicationContext, signatureVerification!!.userEntity, it)
```

Comment:
Compile error
```
> Task :app:compileGplayDebugKotlin FAILED
e: /drone/src/app/src/gplay/java/com/nextcloud/talk/services/firebase/MagicFirebaseMessagingService.kt: (167, 122): Too many arguments for public final fun cancelAllNotificationsForAccount(context: Context?, conversationUser: UserEntity): Unit defined in com.nextcloud.talk.utils.NotificationUtils
```

@nickvergessen I guess you would rather want to use ```cancelExistingNotificationWithId``` but I don't know anything about the notification part in Talk...

Reply:
Fixed, yeah was a copy mistake